gcloud-java-resourcemanager/src/main/java/com/google/gcloud/resourcemanager/Project.java

```diff
@@ -67,7 +68,8 @@ public ResourceManager resourceManager() {
   /**
    * Returns a Project object with updated project information.
    *
-   * @return Project object containing the project's updated metadata
+   * @return Project object containing the project's updated metadata or null if the project no
+   *     longer exists
```

Comment:
`null` could also be returned if the user feeds to the Project class infos of a non-existing project, this is unlikely to happen but I was thinking about phrasing this as: `or {@code null} if not found`